enterprise/server/remote_execution/runner/runner_test.go

```diff
@@ -419,18 +419,20 @@ func TestRunnerPool_DefaultSystemBasedLimits_CanAddAtLeastOneRunner(t *testing.T
 	assert.Equal(t, 1, pool.PausedRunnerCount())
 }
 
+type DiskOnlyCtrProvider struct{}
```

Comment:
nit: carry over the comment here

```suggestion
// Returns containers that only consume disk resources when paused (like firecracker).
type DiskOnlyCtrProvider struct{}
```